morphir/src/org/finos/morphir/runtime/sdk/DictSDK.scala

```diff
@@ -27,4 +27,121 @@ object DictSDK {
         RTValue.Map(newDict)
       }
   }
+
+  val diff = DynamicNativeFunction2("diff") {
+    // === ELM Function ===
+    // diff : Dict comparable a -> Dict comparable b -> Dict comparable a
+    // Keep a key-value pair when its key does not appear in the second dictionary.
+    (ctx: NativeContext) => (dict1Raw: RTValue.Map, dict2Raw: RTValue.Map) =>
+      {
+        val dict1 = dict1Raw.value.clone()
```

Comment:
The use of clone is not needed in this implementation as `filter` will not mutate the original underlying `LinkedHashMap`

Reply:
https://www.scala-lang.org/api/2.13.6/scala/collection/mutable/LinkedHashMap.html#filter(pred:A=%3EBoolean):C

Reply:
fixed